### PR TITLE
Update W.A.R. page

### LINKED ISSUE #517: web_accessible_resources docs are outdated for manifest v3
**Describe the bug**
The web_accessible_resources docs are out of date for manifest v3.  The examples will only work in manifest v2 or v1.

Using the format described in the docs results in the error `Invalid value for 'web_accessible_resources[0]'.  Entry must be a dictionary value.`

https://developer.chrome.com/docs/extensions/mv3/manifest/web_accessible_resources/

Compare with https://stackoverflow.com/questions/66191143/chrome-extension-invalid-value-for-web-accessible-resources0-could-not-loa

### PR DESCRIPTION
**Pull request by awfuchs**
Update [web accessible resources page](https://developer.chrome.com/docs/extensions/mv3/manifest/web_accessible_resources/) for MV3. Fixes #517.

### CODE REVIEW

**Review by dotproto — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/mv3/manifest/web_accessible_resources/index.md [hunk 1]
```diff
@@ -6,29 +6,73 @@ updated: 2018-05-14
...
+Prior to Manifest V2 all resources within an extension could be accessed from any page on the
+web. This allowed a malicious website to [fingerprint][6] the extensions that a user has installed
+or exploit vulnerabilities (for example [XSS bugs][7]) within installed extensions. 
+
+Beginning with Manifest V2, access to those resources was limited to rotect the privacy of users. MV2
+extensions exposed only those resources explicitly designated as web accessible.
+
+Manifest V3 provides finer-grained control, letting you expose individual resources to specified
+pages, domains, or extensions.
+{% endAside %}
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/extensions/mv3/manifest/web_accessible_resources/index.md:27]: I had considered and rejected that, but will keep thinking about this.

File: site/en/docs/extensions/mv3/manifest/web_accessible_resources/index.md [hunk 2]
```diff
@@ -6,29 +6,73 @@ updated: 2018-05-14
...
-    "script/double-rainbow.js",
-    "script/main.js",
-    "templates/*"
+    {
+      "resources": [ "test1.png", "test2.png" ],
+      "matches": [ "https://web-accessible-resources-1.glitch.me/*" ]
+    }, {
+      "resources": [ "test3.png", "test4.png" ],
+      "matches": [ "https://web-accessible-resources-2.glitch.me/*" ],
+      "use_dynamic_url": true
```
[the remark this answers is not in the document]
Comment (solomonkinard) [reply site/en/docs/extensions/mv3/manifest/web_accessible_resources/index.md:45]: sg

**Review by solomonkinard — APPROVED** [review]
lgtm % nit

File: site/en/docs/extensions/mv3/manifest/web_accessible_resources/index.md [hunk 3]
```diff
@@ -6,29 +6,68 @@ updated: 2018-05-14
...
-`chrome-extension://[PACKAGE ID]/[PATH]`, which can be generated with the [extension.getURL][1]
-method. Allowlisted resources are served with appropriate [CORS][2] headers, so they're available
+Each object in the array contains these properties:
+<dl>
+  <dt><code>resources</code></dt>
+  <dd>An array of resources to be exposed. Resources are specified as strings. Directories can be exposed using asterisks (e.g. `/images/*` exposes everything in the images directory).</dd>
+  <dt><code>matches</code></dt>
+  <dd>A list of URL match patterns specifying which pages can access the resources. Only the origin is used to match URLs; subdomains patterns (`*.google.com`) and paths are ignored.</dd>
+  <dt><code>extensions</code></dt>
+  <dd>A list of extension IDs, specifying which extensions can access the resources.</dd>
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/manifest/web_accessible_resources/index.md:59]:
```suggestion
  <dt><code>resources</code></dt>
  <dd>An array of resources to be exposed. Resources are specified as strings and may contain
  <code>*</code> for wildcard matches. For example, <code>"/images/*"</code> exposes everything in
  the extension's /images directory recursively while <code>"*.png"</code> exposes all PNG
  files.</dd>
  <dt><code>matches</code></dt>
  <dd>A list of URL match patterns specifying which pages can access the resources. Only the origin
  is used to match URLs; subdomains patterns (<code>*.google.com<code>) and paths are ignored.</dd>
  <dt><code>extensions</code></dt>
  <dd>A list of extension IDs, specifying which extensions can access the resources.</dd>
```

Replaces backticks with `<code>` tags, wraps at 100 characters, and clarifies wildcard behavior.

**Review by dotproto — APPROVED** [review]
Nit suggestion. We'll need to come back to this page soon anyway.

LGTM!

**Pull request merged**